### DISCUSSION TITLE
Route style in Cruiser

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#836] [Q&A] [answered]
In the Cruiser app the route between start and destination is visualized by an area like type (it does not look like a PathLayer). Is there an example in the VTM library how to render a route in the same way?

**Comment by devemux86** [comment]
What do you mean?

It is a path layer with the route geometry.

**Reply by demiantres** [reply]
Ok, I will check it again. In Cruiser the route has an outline (black border around the blue line).

**Comment by devemux86** [comment, marked as answer]
These are two path layers for the casing and the core, like in the render themes.

**Reply by demiantres** [reply]
Got it - thanks. This means that the path cannot be transparent?

**Comment by devemux86** [comment]
The path can be transparent if there is nothing else below it (or it is transparent too).

**Comment by andreynovikov** [comment]
What do you mean by transparency?
1. Transparent with outline - no.
2. Semitransparent - practically no (implementation is buggy, see https://github.com/mapsforge/vtm/discussions/830)
